docs/src/introduction.md

```diff
@@ -58,6 +59,32 @@ p4est_connectivity_destroy(connectivity)
 ```
 
 
+## [`PointerWrapper`s](@id pointer_wrappers)
+
+As we have seen in the [previous section](@ref translation_guidelines) many functions
+provided by [`p4est`](https://github.com/cburstedde/p4est) return pointers to `struct`s,
+which requires to [`unsafe_load`](https://docs.julialang.org/en/v1/base/c/#Base.unsafe_load)
+the pointer in order to access the underlying data.
+[P4est.jl](https://github.com/trixi-framework/P4est.jl) offers a convenient way to avoid
+using `unsafe_load` whenever it is necessary by using a [`PointerWrapper`](@ref).
```

Comment:
```suggestion
[P4est.jl](https://github.com/trixi-framework/P4est.jl) offers a convenient alternative to
directly using `unsafe_load` by providing the [`PointerWrapper`](@ref) data type.
```
Suggestion: "avoid" sounds to me like it is something bad and our approach is "better"; I'd just say it's an alternative (especially we use `unsafe_load` under the hood).